.github/workflows/ci.yaml

```diff
@@ -57,10 +57,51 @@ jobs:
         run: |
           python -m pytest
 
+  nlp-regression:
+    runs-on: ubuntu-latest
+    env:
+      UMLS_API_KEY: ${{ secrets.UMLS_API_KEY }}
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install Docker
+        uses: docker/setup-buildx-action@v3
+
+      - name: Build ETL image
+        uses: docker/build-push-action@v5
+        with:
+          load: true  # just build, no push
+          tags: smartonfhir/cumulus-etl:latest
+
+      - name: Download NLP images
+        run: docker compose --profile covid-symptom up -d --quiet-pull
```

Comment:
It's not clear how to best cache the pulled images...?  Do you think GitHub does some network-side caching? There are some rando actions in the github action marketplace that promise to cache pulled images... But it might be easy enough to do on our own too.

On the other hand... it's just GitHub's resources we're annoying - which makes me suspect they do some their-side caching, just to save their own money and maybe we don't have to be clever.

Reply:
I would assume they've got something akin to artifactory instance(s) inside their infrastructure. Since it's open source and we don't pay for time, I don't think we need to worry about optimizing - but if we were, it would probably be something similar - add a cache box and update docker hosts to point at it?